app/frontend/styles/_filter.scss

```diff
@@ -30,6 +30,10 @@
   box-shadow: none;
 }
 
+.moj-pagination__list {
+  padding-left: 3px;
+}
+
 .app-filter {
   display: none;
```

Comment:
I know that this isn't part of the review but this means without JS the user won't be able to see filters on small screens which is inaccessible. See prototype code.